handler.go

```diff
@@ -49,12 +51,19 @@ type item struct {
 	Comments []comment `json:"-"`
 }
 
+type user struct {
+	ID        bson.ObjectId `json:"id"`
```

Comment:
should this be `bson:"_id"`? I guess it doesn't matter for indexed objects, but may be good to keep it consistent